main.py

```diff
@@ -0,0 +1,191 @@
+    if info_attribute in search_list:
+        if info_attribute == date_attribute and len(info_value) != 4 and "-" not in info_value:
+            info_value = info_value[:4] + "-" + info_value[4:6] + "-" + info_value[6:]
+        return [info_attribute, info_value]
+    else:
+        return None
+
+
+def get_book_info_template() -> dict:
+    book_info_dictionary = {
+        "Sku": "",
+        "ISBN 13": "",
+        "ISBN 10": "",
+        "Title": "",
+        "Author": "",
+        "Publisher": "",
+        "Year published": "",
+        "Price": ""
+    }
+    return book_info_dictionary
+
+
+def get_template_other_than_book() -> dict:
+    other_info_dictionary = {
+        "Sku": "",
+        "Title": "",
+        "Studio": "",
+        "EAN": "",
+        "Release date": "",
+        "Price": ""
+    }
+    return other_info_dictionary
+
+
+def get_category_info(url: str, is_book: bool) -> Optional[dict]:
+    if is_book:
+        info_dictionary = get_book_info_template()
+    else:
+        info_dictionary = get_template_other_than_book()
+    html = get_response_of_url(url)
+    if html is not None:
+        price_element = find_element_in_soup(html, "div", {"class": "price"}, False)
+        if price_element is not None:
+            info_dictionary["Price"] = price_element.text
+        info_elements = find_element_in_soup(html, "div", {"class": "attributes"}, False)
+        if info_elements is not None:
+            for info in info_elements:
+                try:
+                    label_text = info.label.text
+                    div_text = info.div.text
+                except AttributeError:
+                    continue
+                except Exception as e:
+                    print(e)
+                    continue
+                processed_info = process_info(label_text, div_text, is_book)
+                if processed_info is not None:
+                    info_dictionary[processed_info[0]] = processed_info[1]
+        else:
+            return None
+    return info_dictionary
+
+
+def process_different_category(session_book, database_book: db_book,
+                               session_other, database_other: db_other, search_pages):
+    html = get_response_of_url("https://www.wob.com/en-gb")
+    if html is not None:
+        category_list = find_element_in_soup(html, "div", {"class": "categoryItem"}, True)
+        if category_list is not None:
+            for category in category_list:
+                if category.a is not None:
+                    is_book = True if "Books" in category.a.text else False
+                    if is_book:
+                        process_category(session_book, database_book, "https://www.wob.com" + category.a["href"],
+                                         category.a.text, is_book, search_pages)
+                    else:
+                        process_category(session_other, database_other, "https://www.wob.com" + category.a["href"],
```

Comment:
I would do 
`process_category(session_book if is_book else session_other,
`
or you can put it into a variable, but I wouldn't make this whole call twice